samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -389,16 +400,19 @@ function disableAllFeedbackCheckTemplate(feedbackType)
 	}
 }
 
-$(document).ready(function(){
+$(window).load( function() {
```

Comment:
Forgot to mention that window.load is fired later than document.ready
http://stackoverflow.com/questions/8396407/jquery-what-are-differences-between-document-ready-and-window-load